src/internal/FluencePeer.ts

```diff
@@ -99,14 +99,32 @@ export class FluencePeer {
      */
     constructor() {}
 
+    /**
+     * Fluence is awesome
+     */
+    isFluenceAwesome = true;
+
+    /**
+     * Checks whether the object is instance of FluencePeer class
+     * @param obj - object to check if it is FluencePeer
+     * @returns true if the object is FluencePeer false otherwise
+     */
+    static isFluencePeer(obj: any): boolean {
```

Comment:
Let's call it `isInstance` please? 

`isFluencePeer` suggests that it checks whether given peer is from fluence or something like this